### ISSUE TITLE
DSM model setup 2 ailerons+2 flaps

### ISSUE THREAD

**Issue by silicongarage** [frankiearzu/DSMTools#1]
There is a problem in v0.53 DsmSetupLib.lua not handling the case of 2 ailerons + 2 flaps correctly.
I've edited lines 196 and 790 to reflect using references to the Right flap.  This appears to fix the problem.

**Comment by frankiearzu** [comment]
Hi! Im also in Austin, TX.  Where do you fly? ARCA?
Will check your fix, thank you.
Frankie

**Comment by silicongarage** [comment]
Hi,  first thank you for the effort you  have put in getting DSM forward programming to this state!   I am new to EdgeTX but excited that I can now setup a model without having to resort to my NX.  I've used a TX16S with this code on an AR637TA (unlocked) and an AR8360T successfully.  I've been trying to find someplace close for flying and have recently been going to Wooten Park.

here is a diff of the changes I made
```
196c196
<         MENU_DATA[MEMU_VAR.CH_L_FLP] = PORT.PORT5
---
>         MENU_DATA[MEMU_VAR.CH_R_FLP] = PORT.PORT5
790c790
<             ctx.MenuLines[5] = { Type = LINE_TYPE.LIST_MENU_NC, Text=rightFlapText, TextId = 0, ValId = MEMU_VAR.CH_L_FLP, Min=0, Max=9, Def=0, Val= leftFlap }
---
>             ctx.MenuLines[5] = { Type = LINE_TYPE.LIST_MENU_NC, Text=rightFlapText, TextId = 0, ValId = MEMU_VAR.CH_R_FLP, Min=0, Max=9, Def=0, Val= rightFlap }
```
Buster

**Comment by frankiearzu** [comment]
Fixed on version v0.54
